### ISSUE TITLE
Quarto project does not appear even after updating RStudio

### ISSUE THREAD

**Issue by jooyoungseo** [quarto-dev/quarto-cli#2760]
I am reporting this on behalf of @sharvitomar

> i have been trying to update my RStudio/install Quarto. The process so far has not been successful. RStudio help tab's Check for updates option tells me that the current version is upto date. While installing Quarto from the website, i faced some version conflicts and after resolving I still couldn't find Quarto option in the New Project. I then tried to install the new version of RStudio and R. Currently due to some OS incompatibility, it is crashing.

@sharvitomar -- Please provide your OS and RStudio information.

**Comment by sharvi24** [comment]
**Version information:**
RStudio Version 1.4.1717
R version 3.3.3
macOS version 10.14.6

**Comment by sharvitomar** [comment]
**Version Information**
RStudio Version 1.4.1717
R version 3.3.3
macOS version 10.14.6

**Comment by cscheid** [comment]
> R version 3.3.3
This version of R is about 5 years old. Please upgrade to 4.2.1, the latest stable version of R.

You probably want to install the latest stable version of the RStudio IDE as well.

**Comment by sharvitomar** [comment]
RStudio latest version is compatible with macOS version 10.15+ however my OS version is 10.14.6. The latest version that my OS can support is 2021.09.0 which does not happen to come with Quarto. (Quarto is available in 2022 releases).

I upgraded R to version 4.2.1 however, that did not help with Quarto installation.

What worked out for me was using this hack ([https://community.rstudio.com/t/rstudio-requires-quarto-0-9-80-but-not-available/138913/5]).  So I downloaded the latest release of the quarto from [https://github.com/quarto-dev/quarto-cli/releases](https://github.com/quarto-dev/quarto-cli/releases)
The visual mode/editor working as RStudio relies on pandoc being in the quarto/bin folder but in the new Quarto builds they have put it in the quarto/bin/tools folder. So I copied those files and now it becomes available in IDE under New project options!
